week1/homework/src/server.js

```diff
@@ -5,16 +5,46 @@ const http = require('http');
 /* `createServer` MUST return an instance of `http.Server` otherwise the tests
  * will fail.
  */
+
+function responseHandler(response, status, state) {
```

Comment:
I think `responseHandler` is not the best possible name.  Can you come up with a better name?